include/d/d_attention.h

```diff
@@ -166,15 +205,15 @@ class dAttention_c {
 
     fopAc_ac_c* Owner() { return (fopAc_ac_c*)mpPlayer; }
     
-    bool chkFlag(u32 flag) { return (mFlags & flag) ? true : false; }
-    void setFlag(u32 flag) { mFlags |= flag; }
-    void clrFlag(u32 flag) { mFlags &= ~flag; }
-    bool Lockon() { return LockonTruth() || chkFlag(0x20000000); } // regswap
+    bool chkFlag(AttentionFlags flag) { return (mFlags & flag) ? true : false; }
+    void setFlag(AttentionFlags flag) { mFlags |= flag; }
+    void clrFlag(AttentionFlags flag) { mFlags &= ~flag; }
```

Comment:
Don't change parameter types, we know what the devs used from the debug maps
```suggestion
    bool chkFlag(u32 flag) { return (mFlags & flag) ? true : false; }
    void setFlag(u32 flag) { mFlags |= flag; }
    void clrFlag(u32 flag) { mFlags &= ~flag; }
```